internal/pkg/cli/env_delete.go

```diff
@@ -173,6 +183,20 @@ func (o *deleteEnvOpts) Execute() error {
 	}
 	o.prog.Stop(log.Ssuccessf(fmtRetainEnvRolesComplete, o.name))
 
+	o.prog.Start(fmt.Sprintf("Emptying S3 buckets managed by the %q environment\n", o.name))
+	if err := o.emptyBuckets(); err != nil {
+		o.prog.Stop(log.Serrorf("Failed to empty buckets managed by the %q environment\n", o.name))
+		switch t := err.(type) {
+		// Handle error and recommend action, don't exit program
+		case *errBucketEmptyingFailed:
+			log.Errorln(t.Error())
+			log.Warningln(t.RecommendActions())
+		default:
+			return err
```

Comment:
I wonder if we should just swallow this error and move on to deleting the stack 🤔  For example maybe some API calls in `emptyBuckets` got transient error, and the stack deletion could have still succeeded.

Reply:
Reviewing the possible errors you can get, I agree with this. I'm changing the way the error is handled for it.